bundle/compliance/policy/aws_config/ensure_config_enabled.rego

```diff
@@ -0,0 +1,25 @@
+package compliance.policy.aws_config.ensure_config_enabled
+
+import data.compliance.lib.common as common
+import data.compliance.policy.aws_config.data_adapter
+import future.keywords.every
+
+default rule_evaluation = false
+
+rule_evaluation {
+	# every config needs to have at least 1 enabled recorder
+	every config in data_adapter.configs {
+		recorder := config.recorders[_]
+		recorder.ConfigurationRecorder.RecordingGroup.AllSupported == true
+		recorder.ConfigurationRecorder.RecordingGroup.IncludeGlobalResourceTypes == true
+	}
```

Comment:
from the rule audit description: 
> Evaluate the output to ensure that there's at least one recorder for which recordingGroup object includes "allSupported": true AND
      "includeGlobalResourceTypes": true
      
combined with the title: 
> Ensure AWS Config is enabled in all regions

so the rego rule takes *ALL* configs from all regions and verifies *ALL* of them have at least 1 recorder enabled.